File: koin-aliansi-code-challenge/app/src/main/java/id/niteroomcreation/code/domain/model/DataResponse.java
package id.niteroomcreation.code.domain.model;

import java.util.List;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class DataResponse{

	public static final String DATA_IMAGE = "image";
	public static final String DATA_CONTACT = "contact";

	@SerializedName("data")
	@Expose
	private List<DataItem> data;

	public List<DataItem> getData(){
		return data;
	}

	@Override
	public String toString() {
		return "DataResponse{" +
				"data=" + data +
				'}';
	}
}

File: app/src/main/java/id/test/myapplication/model/DataResponse.kt
package id.test.myapplication.model


import com.google.gson.annotations.SerializedName

data class DataResponse(
    @SerializedName("data")
    val `data`: List<Data?>?
) {
    data class Data(
        @SerializedName("attachment")
        val attachment: String,
        @SerializedName("body")
        val body: String,
        @SerializedName("from")
        val from: String?,
        @SerializedName("id")
        val id: Int?,
        @SerializedName("timestamp")
        val timestamp: String?,
        @SerializedName("to")
        val to: String?
    )
}